src/simpleNbox.cpp

```diff
@@ -408,89 +478,140 @@ unitval SimpleNbox::getData(const std::string& varName,
                             const double date) throw ( h_exception )
 {
     unitval returnval;
+
+    std::string biome = SNBOX_DEFAULT_BIOME;
+    std::string varNameParsed = varName;
+    std::string biome_error = "Requested biome missing from biome list. "
+        "Hit this error while trying to retrieve variable: '" + varName + "'.";
```

Comment:
I think I originally removed the `+` because I was hitting an error that concatenating string literals via `+` was not defined (basically, the problem described [here](https://stackoverflow.com/questions/6061648/concatenate-two-string-literals)). However, adding it back does not throw an error, so I guess that was a different situation 🤷‍♂ 

The corresponding change was in f4d8f1cf57272ddbb5725596d1e40c21ef22a966.